server/services/applicationService.test.ts

```diff
@@ -393,6 +393,15 @@ describe('ApplicationService', () => {
         status: 'assesmentInProgress',
       })
     })
+
+    it('parameters not passed then calls the client with default values for page no and search and sort options', async () => {
```

Comment:
I think perhaps the key point we need to include here is that that the default params give us our applications **ordered newest first**?